src/api/admin.ts

```diff
@@ -0,0 +1,61 @@
+import { DB, ServiceKey } from './types/admin'
+import { setData, getData, data } from './lib/data'
+import express from 'express'
+import fs from 'fs'
+
+const app = express.Router()
+
+const serviceKeys = getData('serviceKeys')
```

Comment:
remove, serviceKeys might not be ready yet when the app first runs. You should get the serviceKeys on every request.